main.go

```diff
@@ -59,12 +61,23 @@ func main() {
 		configMapRepl.Run()
 	}()
 
+	go func() {
```

Comment:
Would you mind to call these functions directly and not in an anonymous function? Like:  
```go
go roleRepl.Run()
```

I know, we've done this before but I can't figure out  a valid reason for that :3

It would be great if you'd also adjust the other anonymous calls above (lines 57 and 61)